src/App.tsx

```diff
@@ -1,73 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
-import { peopleFromServer } from './data/people';
+import { Dropdown } from './components/Dropdown';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+
+  const handlePersonTitleChange = (person: Person | null) => {
```

Comment:
FYI: it's redundant function wrapper - you can just use `setSelectedPerson` instead of `handlePersonTitleChange`